src/wallet/service_test.go

```diff
@@ -422,7 +422,7 @@ func (dug dummyUnspentGetter) Get(uxid cipher.SHA256) (coin.UxOut, bool) {
 func TestServiceCreateAndSignTx(t *testing.T) {
 	dir := prepareWltDir()
 
-	s, err := NewService(dir)
+	s, err := NewService(dir, false)
```

Comment:
Need test with disableWalletAPI=true

Reply:
@gz-c ,
Added tests with disableWalletAPI=true.
Skipped TestServiceCreateAndSignTx test, because logic changes with DisableWalletAPI=true.
Added return with ErrWalletApiDisabled for CreateWallet, because got panic without DisableWalletAPI=true param check.
I can add this check for all wallet service methods if you don't mind.